### PR TITLE
Add ability to tag nodes for further usage in constraints

### PR DESCRIPTION
**Pull request by maxbr**
### Short description:

I found the following issue(s) or bug(s):
  - 

_And/OR_

I found that the following _enhancement_ is worthwhile:
  - 

### Bookkeeping
I did the following bookkeeping:
 - [ ] If a major change, bumped the playbook version in `vars/`
 - [ ] Added as much documentation as I could (we're not looking a book, just enough to help others)
 - [ ] Confirmed that the PR works by running the vagrant tests (if any) and/or added to the tests (see `ci/` or legacy `tests/`)
 - [ ] Followed up with any automated test results as a result of the pull request

### CODE REVIEW

File: handlers/main.yml [hunk 1]
```diff
@@ -13,3 +13,6 @@
 - name: Restart mesos-slave
   service: name=mesos-slave state=restarted
   when: mesos_install_mode == "slave" or mesos_install_mode == "master-slave"
+
+- name: Clear mesos-slave cache
+  file: path=/var/lib/mesos/meta/slaves/latest state=absent
```
Comment (ernestas-poskus) [inline handlers/main.yml]: add new line

File: defaults/main.yml [hunk 2]
```diff
@@ -28,6 +28,7 @@ zookeeper_hostnames: "{{ mesos_hostname }}:{{ zookeeper_client_port }}"
 mesos_zookeeper_masters: "zk://{{ zookeeper_hostnames }}/mesos"
 mesos_owner: root
 mesos_group: root
+mesos_node_type: general
```
Comment (ernestas-poskus) [inline defaults/main.yml]: I assume this is default value ? can you provide documentation as required in PR description ?

File: templates/conf-mesos-slave.j2 [hunk 3]
```diff
@@ -5,6 +5,7 @@ export MESOS_CONTAINERIZERS="{{ mesos_containerizers }}"
 export MESOS_EXECUTOR_REGISTRATION_TIMEOUT="{{ mesos_executor_timeout }}"
 export MESOS_PORT="{{ mesos_slave_port }}"
 export MESOS_WORK_DIR="{{ mesos_work_dir }}"
+export MESOS_ATTRIBUTES="node_type:{{ mesos_node_type }}"
```
Comment (ernestas-poskus) [inline templates/conf-mesos-slave.j2]: can you provide documentation as required in PR description ?

File: handlers/main.yml (hunk 1 above)
Comment (veger) [inline handlers/main.yml]: This is probably the solution for https://github.com/AnsibleShipyard/ansible-mesos/issues/65 :)

File: templates/conf-mesos-slave.j2 (hunk 3 above)
Comment (veger) [inline templates/conf-mesos-slave.j2]: Setting attributes is currently done using:
```
mesos_slave_additional_configs:
    - name: ATTRIBUTES
      value: '"rack:external"'
```

Does this change break this original behavior?

File: handlers/main.yml (hunk 1 above)
Comment (ernestas-poskus) [inline handlers/main.yml]: Yes this should solve the issue

**Pull request closed without merging**